packages/samples/mvp-dapp/src/omnilock/omnilock.model.ts

```diff
@@ -64,4 +67,9 @@ export class OmnilockModel extends JSONStore<Record<string, never>> {
       witnesses: [],
     }
   }
+
+  #filterCells = () => {
+    // TODO: is this filter necessary?
```

Comment:
It has been filtered by `store` with cellPattern. So it's unnecessary.

Reply:
The filter has been removed by https://github.com/ckb-js/kuai/pull/157/commits/06e4bc2952c1a45251a5f37e562cdb1896aea97b